jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/coder/OdeBsmDataCreatorHelperTest.java

```diff
@@ -18,16 +15,16 @@
partII-Value><VehicleSafetyExtensions><events>1000000000000</events><pathHistory><crumbData><PathHistoryPoint><latOffset>-1347</latOffset><lonOffset>-3753</lonOffset><elevationOffset>22</elevationOffset><timeOffset>61530</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>-1998</latOffset><lonOffset>-6328</lonOffset><elevationOffset>-12</elevationOffset><timeOffset>62370</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>-1887</latOffset><lonOffset>-8337</lonOffset><elevationOffset>-39</elevationOffset><timeOffset>62710</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>-1166</latOffset><lonOffset>-10100</lonOffset><elevationOffset>-57</elevationOffset><timeOffset>63020</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>164</latOffset><lonOffset>-11501</lonOffset><elevationOffset>-77</elevationOffset><timeOffset>63380</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>2130</latOffset><lonOffset>-12445</lonOffset><elevationOffset>-95</elevationOffset><timeOffset>63780</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>4586</latOffset><lonOffset>-12737</lonOffset><elevationOffset>-102</elevationOffset><timeOffset>64220</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>6499</latOffset><lonOffset>-12139</lonOffset><elevationOffset>-111</elevationOffset><timeOffset>64670</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>8041</latOffset><lonOffset>-10872</lonOffset><elevationOffset>-90</elevationOffset><timeOffset>65030</timeOffset></PathHistoryPoint><PathHistoryPoint><latOffset>10185</latOffset><lonOffset>-7426</lonOffset><elevationOffset>-53</elevationOffset><timeOffset>65535</timeOffset></PathHistoryPoint></crumbData></pathHistory><pathPrediction><radiusOfCurve>32767</radiusOfCurve><confidence>200</confidence></pathPrediction><lights>000010000</lights></VehicleSafetyExtensions></partII-Value></PartIIcontent><PartIIcontent><partII-Id>2</partII-Id><partII-Value><SupplementalVehicleExtensions><classification>0</classification><classDetails><keyType>0</keyType><hpmsType><none/></hpmsType></classDetails><vehicleData/><doNotUse1><isRaining><error/></isRaining><rainRate>65535</rainRate><precipSituation><unknown/></precipSituation><solarRadiation>65535</solarRadiation><friction>101</friction><roadFriction>0</roadFriction></doNotUse1><doNotUse2><airTemp>73</airTemp><airPressure>41</airPressure><rainRates><statusFront><washerInUse/></statusFront><rateFront>0</rateFront></rainRates></doNotUse2></SupplementalVehicleExtensions></partII-Value></PartIIcontent><PartIIcontent><partII-Id>1</partII-Id><partII-Value><SpecialVehicleExtensions><vehicleAlerts><doNotUse>0</doNotUse><sirenUse><notInUse/></sirenUse><lightsUse><notInUse/></lightsUse><multi><unavailable/></multi><events><doNotUse>0</doNotUse><event>1000000000000000</event></events></vehicleAlerts></SpecialVehicleExtensions></partII-Value></PartIIcontent></partII></BasicSafetyMessage></value></MessageFrame></data></payload></OdeAsn1Data>";
```

Comment:
We should separate the XM Lin this file into an individual XML file and make sure it includes all possible optional values from the schema.

Reply:
Addressed